translations/ui-data-import/en.json

```diff
@@ -236,6 +236,7 @@
   "validation.quotationError": "Non-MARC value must use quotation marks",
   "validation.cannotBeUpdated": "This field cannot be updated",
   "validation.chooseDifferentField": "Please choose a different field",
+  "validation.selectAnAction": "Action must be selected before the profile can be saved",
```

Comment:
Are you sure the validation message should look like this? In the ticket, I see "Select an action" warning.

Reply:
It's ok, discussed with Ann-Marie, and updated the description of the ticket in JIRA.